pkg/server/plugin/nodeattestor/k8s/sat/sat.go

```diff
@@ -187,24 +195,23 @@ func (p *AttestorPlugin) Attest(stream nodeattestorv1.NodeAttestor_AttestServer)
 		if err != nil {
 			return err
 		}
-
-		// TODO: service account tokens don't currently expire.... when they do, validate the time (with leeway)
-		if err := claims.Validate(jwt.Expected{
-			Issuer: "kubernetes/serviceaccount",
-		}); err != nil {
-			return status.Errorf(codes.InvalidArgument, "unable to validate token claims: %v", err)
-		}
-
-		if claims.Namespace == "" {
-			return status.Error(codes.InvalidArgument, "token missing namespace claim")
+		if !claims.Expiry.Time().IsZero() {
+			// This is an indication that this may be a projected service account token
+			p.log.Warn(`The service account token has an expiration time, which is an indication that may be a projected service account token.
```

Comment:
How does this message show up in the logging? the  `` literal includes a newline and a bunch of tabs. Should we remove those?

Reply:
I'll remove those special characters.